context/app/static/js/components/workspaces/WorkspaceEnvironmentDescription/WorkspaceEnvironmentDescription.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { PropsWithChildren } from 'react';
+import Typography from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+
+import ContactUsLink from 'js/shared-styles/Links/ContactUsLink';
+
+function HighEmphasis({ children }: PropsWithChildren) {
+  return (
+    <Typography variant="subtitle2" color="textPrimary" component="span">
+      {children}
+    </Typography>
+  );
+}
```

Comment:
This could be moved to `shared-styles` - not sure if bolded text is a pattern elsewhere?